### DISCUSSION TITLE
github.com: "You have triggered a rate limit"

### DISCUSSION THREAD

**Discussion by sevfurneaux** [community/community#37828] [Other Feature Feedback, Questions, & Ideas]
When I'm on my WiFi and viewing any URL on github.com, I am receiving this page:

<img width="583" alt="Screenshot 2022-10-30 at 22 33 24" src="https://user-images.githubusercontent.com/1056416/198905169-f100f6d6-a312-4442-af2a-f67a53408f83.png">

...seems fine on 4g mobile though.

Has a specific I.P been restricted for my account?

**Comment by ghost** [comment]
I saw the same for several hours on both laptops. Ok on mobile apps, Android and iOS. Seems to be ok now. All status pages that I could see said everything was working normally. Strange as I had hardly used Github all day so there was no way that I had triggered the rate limit.

**Comment by ossanna16** [comment]
Hi there @sevfurneaux and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
